api/response/schema.py

```diff
@@ -1,17 +1,28 @@
 from datetime import datetime
 import graphene
-from helpers.auth.authentication import Auth
 from graphene_sqlalchemy import SQLAlchemyObjectType
-from api.response.models import Response as ResponseModel
-from utilities.validations import validate_empty_fields
 from graphql import GraphQLError
-from api.room.schema import Room
 from api.question.models import Question as QuestionModel
+from api.response.models import Response as ResponseModel
+from api.room.schema import Room
+from helpers.auth.authentication import Auth
 from helpers.pagination.paginate import ListPaginate
-from helpers.response.create_response import (
-    ResponseData, ResponseDetail, map_response_type,
-    create_response, create_response_details
-)
+from helpers.response.create_response import (MissingItems, Rate,
+                                              SelectedOptions, TextArea,
+                                              create_response,
+                                              create_response_details,
+                                              map_response_type)
+from utilities.utility import update_entity_fields
+from utilities.validations import validate_empty_fields
+
+
+class ResponseData(graphene.Union):
+    class Meta:
+        types = (Rate, SelectedOptions, TextArea, MissingItems)
+
+    @classmethod
+    def resolve_type(cls, instance, info):
+        return type(instance)
```

Comment:
The same Object is in `create_response.py` try and reuse the code to avoid identical blocks